docs/beaver-builder/getting-started/bb-editor-basics/launch-builder.md

```diff
@@ -56,8 +45,7 @@ other methods described here to open the Beaver Builder editor.
 
 ## 2 From the **All Pages** list
```

Comment:
You'll want to remove the `**` from **All Pages** as it messes with the TOC markup.

Reply:
@nancyhild Did you see the above! :D

Reply:
Ah, fixed it, thanks.